deps/ncrypto/ncrypto.cc

```diff
@@ -3080,6 +3080,69 @@ bool CipherCtxPointer::getAeadTag(size_t len, unsigned char* out) {
 
 // ============================================================================
 
+KdfCtxPointer KdfCtxPointer::FromName(std::string_view name) {
```

Comment:
This will need to be ifdef guarded so it only compiles when openssl/boringssl versions that support `EVP_KDF`